### PR TITLE
UI GPU Checker!

### PR DESCRIPTION
**Pull request by 5he1n**
Yet another UI checker

Already with Russian loc

### CODE REVIEW

File: server_release_template/web_gui/js/app/setupWizard.js [hunk 1]
```diff
@@ -7,7 +7,39 @@ define([
...
+                const gl = document.createElement('canvas').getContext('webgl');
+                const debugInfo = gl.getExtension("WEBGL_debug_renderer_info");
+                var rawGPUInfo = gl.getParameter(debugInfo.UNMASKED_RENDERER_WEBGL);
+
+                this.fullName = rawGPUInfo.match(/((NVIDIA|AMD|Intel)[^\d]*[^\s]+)/)[0];
+                [this.dev, this.name] = this.fullName.split(/(?<=^\S+)\s/);
+            }
+        }
+
+        function CheckGPUSupport(name) {
```
Comment (zarik5) [inline server_release_template/web_gui/js/app/setupWizard.js:21]: Single letter difference between function names is bad, you can put this into CheckGPUSSupport()

File: server_release_template/web_gui/js/app/nls/wizard.js [hunk 2]
```diff
@@ -28,7 +28,10 @@ define({
         "driverFailed": "Registering driver failed",
         "driverSuccess": "Driver successfully registered",
         "firewallFailed": "Setting firewall rules failed",
-        "firewallSuccess": "Firewall rules successfully set"
+        "firewallSuccess": "Firewall rules successfully set",
+        "YourGPUIs": "Your GPU is",
+        "GPUSupported": "Great! This GPU may be supported!",
```
Comment (zarik5) [inline server_release_template/web_gui/js/app/nls/wizard.js:33]: "may be" -> "is probably"

File: server_release_template/web_gui/js/app/setupWizard.js [hunk 3]
```diff
@@ -7,7 +7,39 @@ define([
...
+            switch (GPU.dev) {
+                case "Intel(R)":
+                    return '🔴 '+i18n.GPUUnsupported;
+                    break;
+                case "NVIDIA":
+                case "AMD":
+                    return CheckGPUSupport(GPU.name);
+                    break;
+                default:
+                    return "🟣 WT..?! You wanna say you have no GPU? Are you playing on potato?"
```
Comment (zarik5) [inline server_release_template/web_gui/js/app/setupWizard.js:39]: I know this is funny but actually users that see this message may (most probably) have their GPU disconnected. Also add this as a localization string.

File: server_release_template/web_gui/js/app/nls/wizard.js [hunk 4]
```diff
@@ -28,7 +28,10 @@ define({
...
+        "GPUUnsupported": "We are sorry, but this card may be unsupported"
```
Comment (zarik5) [inline server_release_template/web_gui/js/app/nls/wizard.js:34]: Append: "You can try ALVR anyway and see if it works".

**Review by zarik5 — CHANGES_REQUESTED** [review]

**Comment by 5he1n** [comment]
@zarik5 fixed

**Review by zarik5 — APPROVED** [review]

**Pull request merged**